frontend/src/stories/Introduction.stories.mdx

```diff
@@ -4,12 +4,24 @@ import { Meta } from '@storybook/addon-docs';
 
 # Design System
 
-At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation in Angular or Rails.
+At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
 
 Our goal is to reach a point where every view in OpenProject is built with elements described in the design system. However, we are aware that reaching this goal will take time for a tool as complex and layered as OpenProject.
 
 As such, the design system is still in its infancy and will grow with each release.
 
+## Purpose (what it is and what it's not)
+
+This Storybook-based design system documentation only seeks to describe the actual reusable components; it does not seek to document features where these components are used.
+
+The purpose of this document is to:
+
+- Have a single point of truth on what reusable components should look like, how they should behave, and what options they provide.
+- Allow product designers and front-end developers to have a shared understanding of how these common elements are specified and implemented
+- Allow QA a common base of reference when testing implementation of features using these common elements
+- Give new or outside developers important context about available components and how to use them
+
+
 ## SPOT
```

Comment:
As stated above, I struggle with the concept of "single point of truth". As much as I support standardization I would not go so far and call it a "single point of truth". Our products can and should be inconsistent in specific use cases.

Reply:
We came up with this to make clear that neither WPs nor Figma have the final say on reusable components, styles, and patterns. Instead, by documenting something and implementing it in code you can apply a PR review process to it (e.g. both one designer and one dev need to approve changes). This should make the contents of Storybook / the DS docs the best-vetted resource, thus the primary tool to use when to decide what UI elements to use and how to use them.

Of course you'll have specific one-off instances of UI. For example, there'll always only be one app header or one global search bar, but even those should use the same colors, typo and spacings as the rest of the app.

I'm fine with renaming tho, SPOT was chosen / stayed in use because it was the one name out of all the ideas that had the least resistance among the team.